src/AppInstallerCLICore/ConfigurationCommon.cpp

```diff
@@ -54,7 +54,7 @@ namespace AppInstaller::CLI
 
     namespace Configuration
     {
-        void ValidateCommonArguments(Execution::Args& execArgs)
+        void ValidateCommonArguments(Execution::Args& execArgs, bool requireConfigurationSetChoice)

```

Comment:
Since I made it an optional parameter defaulting to false, I didn't need to update both `ConfigureValidateCommand` and `ConfigureExportCommand`, which still call it.